modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/bulk/AutomationBulkActionUi.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright 2021 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     bdelbosc
+ */
+package org.nuxeo.ecm.automation.core.operations.services.bulk;
+
+/**
+ * Bulk Action that runs an automation operation dedicated for UI
+ *
+ * @since 11.5
+ */
+public class AutomationBulkActionUi extends AutomationBulkAction {
+    public static final String ACTION_NAME = "automationUi";
```

Comment:
Shouldn't it be `automation-ui` as in the descriptor?
See https://github.com/nuxeo/nuxeo/pull/4770/commits/b977772499b387f9d1554b2adc420dd49ff689c4#diff-3db1a327a46d448f905eeac9f9d91c6596400e8d1127c136b08ff6d0207d57e1R10.

Or maybe not because there we're referring to the inputStream suffix :thinking:

Reply:
action name is a bulk service concept defined by configuration only used when creating a bulk command.
here it is the name of the computation (used to create metrics and consumer group), the full name of the computation has a namespace prefix and and both are usually camelCase, not easy I agree